dynamic_rest/viewsets.py

```diff
@@ -337,4 +337,19 @@ def list_related(self, request, pk=None, field_name=None):
 
 
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
-    pass
+
+    ENABLE_BULK_CREATION = dynamic_settings.get('ENABLE_BULK_CREATION', True)
```

Comment:
Ah, don't worry about it then; the settings wrapper was recently introduced to make it easier to document our settings. I must have missed this file during the conversion. I can go over it later and change the syntax